app/assets/javascripts/client/inventories/Inventory.js

```diff
@@ -24,6 +24,27 @@
       'districtProductEntries': {
         method: 'GET',
         url: UrlService.url('inventories/:inventory_id/district_product_entries')
+      },
+      'markComplete': {
+        method: 'POST',
+        url: UrlService.url('inventories/:inventory_id/mark_complete'),
+        params: {
```

Comment:
sorry for the lack of details in my original comment:
I don't think we need a new endpoint for this as PATCH'ing the inventory resource would be enough. We just need to update the inventory with that given atribute to modify it's state and therefore perform the desired action.

So instead of `POST /inventories/:inventory_id/mark_complete` (RPC style) we could be doing `PATCH /inventories/:inventory_id` with body `submitted_at=12-04-2015`. If you don't trust the client to set the date, you can still receive `submitted_at=true` and override it internally with `Time.now` and still be perfectly RESTful.

Same with `PUT /inventories/:inventory_id/save_response`(RPC style), it should have been `POST /inventories/:inventory_id/responses`, and since it's idempotent as there should be only one response per user, it could have been `POST /inventories/:inventory_id/responses/:user_id`. Even ``PATCH /inventories/:inventory_id` with body `replied=true` could do the trick in this case. 

Again, these are just suggestions and I would consider them an improvement, it works as it is.